docs/cloud-soar/cloud-soar-apis.md

```diff
@@ -8,27 +8,27 @@ description: Learn how to access Cloud SOAR APIs and API documentation.
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import ApiIntro from '../reuse/api-intro.md';
 
-<!-- Add this once all customers are moved to Cloud SOAR Delivery 2, and remove all the documentation below for "incmansuite_ng:
-
 The Cloud SOAR APIs allow you to manage incidents, triage, and other Cloud SOAR features.
 
-<ApiIntro/>
-
-import CloudSoarApi from '../reuse/csoar-api-table.md';
+:::info
+This article applies to your organization *only if* your instance URL is a subdomain of `soar.sumologic.com`.
+Otherwise, please refer to:
```

Comment:
```suggestion
This article only applies to organizations with URL instance that have a subdomain of `soar.sumologic.com`. If it doesn't, refer to:
```

Reply:
Thanks for the review.
I rewrote the sentences to be similar and to better clarify the situation.
To add more context:
We have customers with "legacy" Cloud SOAR instances that have been provisioned with URLs having a subdomain of "soar.sumologic.com". These customers could have standard Sumo Logic instances with regional and custom alias or could not have any Sumo Logic instance at all. So the rule to identify which part of the document applies to "Delivery 1" instances is to check if the Cloud SOAR application has a URL in the browser URL bar matching the "*.soar.sumologic.com" pattern (despite the their actual Sumo Logic organization URL).

Hope that the new version of the text is better from your point of view.
Thanks